src/controllers/resume.submission.controller.ts

```diff
@@ -98,7 +98,10 @@ const setWebApplicationData = (session: Session, appData: ApplicationData, trans
     appData[WhoIsRegisteringKey] = WhoIsRegisteringType.AGENT;
   }
 
-  generateTrusteeIds(appData);
+  if (isActiveFeature(config.FEATURE_FLAG_ENABLE_TRUSTS_WEB)) {
```

Comment:
we'd need to keep the generateTrustIds so it gets called if we switch the feature flag off, so have on/off = new/old

Reply:
We only need the trustee ids when we have the CHS Screens since they are used to support this. The spreadsheet solution does NOT need them (and they are not in the API Model)

Reply:
ah ok, i thought it was something added for the spreadsheets, i can see it was added recently by looking at the git blame.